src/mscorlib/shared/System/StringComparer.cs

```diff
@@ -94,8 +94,18 @@ public static StringComparer Create(CultureInfo culture, bool ignoreCase)
             {
                 throw new ArgumentNullException(nameof(culture));
             }
+            
+            return new CultureAwareComparer(culture, ignoreCase ? CompareOptions.IgnoreCase : CompareOptions.None);
+        }
+
+        public static StringComparer Create(CultureInfo culture, CompareOptions options)
+        {
+            if (culture == null)
+            {
+                throw new ArgumentException(nameof(culture));
```

Comment:
I know this PR is extremely old but why doesn't this throw an `ArgumentNullException`?

Reply:
@TylerBrinkley I think it is a mistake. Thanks for pointing it out. Generally it's best to open a new issue (or just PR) since posting on closed issues/PR's sometimes gets overlooked.

Reply:
Did somebody follow-up on this? Was an issue created?

Reply:
I created #26570 to address this.